content/id/examples/application/deployment-scale.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
```

We can translate the description also.
```suggestion
apiVersion: apps/v1 # untuk versi sebelum 1.9.0 gunakan apps/v1beta2
```